### ISSUE TITLE
Still maintened ?

### ISSUE THREAD

**Issue by harlock974** [wayoda/LedControl#27]
Hello Wayoda
Thanks for the great job. Ledcontrol library is incredibly useful (and I do love LED displays).
Is the library still maintened ?
I notice that interesting commits didn't received an answer since a long time.
All the best.

**Comment by gordoste** [comment]
Not really. I have created a fork with what I think is the best of people's contributions.
* ESP8266 support
* Ability to choose between hardware and software SPI

**Comment by ExileFox** [comment]
> Not really. I have created a fork with what I think is the best of people's contributions.
> 
> * ESP8266 support
> * Ability to choose between hardware and software SPI

I would like to see a function to get the state of a segment/led added.

**Comment by gordoste** [comment]
If you submit a patch to my fork I will happily accept it.
https://github.com/gordoste/LedControl
